src/components/Autocomplete.vue

```diff
@@ -301,8 +302,21 @@ export default {
      * @return {Array}
      */
     setResults (response) {
-      if (this.resultsProperty && response[this.resultsProperty]) {
-        return response[this.resultsProperty]
+      if (this.resultsProperty) {
```

Comment:
if there are only two conditions, it's better to use if else, easier to read

http://webcache.googleusercontent.com/search?q=cache%3Ahttps%3A%2F%2Fwww.oreilly.com%2Flibrary%2Fview%2Fhigh-performance-javascript%2F9781449382308%2Fch04s02.html&oq=cache%3Ahttps%3A%2F%2Fwww.oreilly.com%2Flibrary%2Fview%2Fhigh-performance-javascript%2F9781449382308%2Fch04s02.html&aqs=chrome..69i57j69i58.1131j0j4&sourceid=chrome&ie=UTF-8